src/services/bee.tsx

```diff
@@ -0,0 +1,111 @@
+import axios, { AxiosInstance, AxiosResponse } from 'axios';
+import { Bee } from "@ethersphere/bee-js"
+
+const bee = new Bee(`${process.env.REACT_APP_BEE_HOST}`)
+
+const beeApiClient = (): AxiosInstance => {
+    return axios.create({
+        baseURL: process.env.REACT_APP_BEE_HOST
+    })
+}
+
+const beeDebugApiClient = (): AxiosInstance => {
+    return axios.create({
+        baseURL: process.env.REACT_APP_BEE_DEBUG_HOST
+    })
+}
```

Comment:
Could this be replaced by `bee-js` calls on top of `Bee` resp `BeeDebug` class?

Reply:
Yeah Im just getting familar with beeJS so I wasn't too sure of all the functionality. I just realized all the types for bee were already in the library! Should have read through the documentation more haha.

I removed the beeJS api client but kept the debug one though since I dont see all the APIs for the debug on beeJS.

Im leaving it wrapped in the "service" for now because I wasnt sure how stable the beeJS library API was and think its easier to update API changes from the single place.  I think once the beeJS library is stable and out of beta this can definitely be removed.